### DISCUSSION TITLE
I must have broke it

### DISCUSSION THREAD

**Discussion by ghost** [Schneegans/Burn-My-Windows#136] [Q&A] [answered]
So first and foremost; this is hands down of my favorite extensions available to GNOME! 

I recently did an Arch install and after getting set up I installed all my extensions; including this one. 

I started geeking out when I saw that Burn My Windows applied it's affects in cooperation with the extension "ddterm", my drop down terminal extension! Man was I excited! 

But suddenly, today I noticed that ddterm was no longer dropping down with the effects of Burn My Windows. I'd gone through all the settings on ddterm to see if I some how broke it; as I believe it happened when modifying ddterminal's "animations". Leading me to believe that there may be a conflict between the two animation sequences. 

Unfortunately, I'm a bit of a noob; so I don't quite know where to locate any log file that may indicate why Burn My Windows is no longer applying it's animations to ddterm. 

Was hoping that posting my inquiry and request for guidance here may lead to a resolution =). I appreciate any suggestions and guidance!

**Comment by Schneegans** [comment, marked as answer]
I just installed `ddterm` on my machine, and everything seems to work. In fact, I played with the animation settings and could not find a way which does not work :smile: 

So the only potential solution which comes to my mind is resetting all settings of `ddterm` (and maybe also of `burn-my-windows`) and see if this changes anything. This can be done with the following commands:

```bash
# First create a backup
dconf dump /com/github/amezin/ddterm/ > ~/ddtermbackup.txt

# Then reset all settings to their defaults
dconf reset -f /com/github/amezin/ddterm/

# Later you could load the backup
dconf load /com/github/amezin/ddterm/ < ~/ddtermbackup.txt
```

You can do the same thing with `burn-my-windows` by replacing `/com/github/amezin/ddterm/` with `/org/gnome/shell/extensions/burn-my-windows/` in the commands above.

**Reply by ghost** [reply]
@Schneegans you are a SAINT! I really appreciate you taking the time to not only help me, but educate me! I couldn't find any "rest to default" options on ddterm and was getting a bit discouraged. 

Thank you so much for helping me resolve this issue! I only needed to reset ddterm and animations resumed and my system is epic once again!
